src/client/pages/Debugger/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const ActualDebugger = React.lazy(() =>
+  import(/* webpackChunkName: "debugger" */ './ActualDebugger').then((m) => ({
+    default: m.ActualDebugger,
+  })),
+)
+
+export const Debugger: React.FC = () => {
+  const [mounted, setMounted] = React.useState(false)
+  React.useEffect(() => setMounted(true), [])
```

Comment:
isn't react suspense doing this logic for us tho?

Reply:
true, we can remove that all together